Source/JavaScriptCore/parser/Parser.cpp

```diff
@@ -897,6 +899,7 @@ template <class TreeBuilder> TreeExpression Parser<LexerType>::parseVariableDecl
 
         failIfTrue(match(PRIVATENAME), "Cannot use a private name to declare a variable");
         if (matchSpecIdentifier()) {
+            failIfTrue(isArgumentsToken() && !findClosetFunctionScope(), "Class static block cannot use 'arguments' as an identifier");
```

Comment:
Is it possible to use `await`?

Reply:
Yes, we can use `await` key word or use `await` as identifier under function scope, which is similar to the usage rule of `arguments` in static block.